src/main/java/frc/team1816/robot/commands/LeftAutoStartCommand.java

```diff
@@ -47,17 +46,17 @@ public void selectAuto() {
         } else if (target == 'R'){
             System.out.println("Switch ---- Target: R");
 
-            addParallel(new SetElevatorHeightPercentCommand(20));
+            addParallel(new RaiseCollectorClawCommand(),1);
+            addParallel(new SetElevatorHeightPercentCommand(20),2);
```

Comment:
Have we tested a timeout of 2 seconds.  This seems like a lot of time if we want 20%.